components/external_server/src/routes/subject.py

```diff
@@ -119,18 +118,14 @@ def post_subject_attribute(subject_uuid: SubjectId, subject_attribute: str, data
 
 @bottle.get("/api/v3/subject/<subject_uuid>/measurements", authentication_required=False)
 def get_subject_measurements(subject_uuid: SubjectId, database: Database):
-    """Return all measurements for the subject within the last 28 weeks."""
+    """Return all measurements for the subject between the date of the report and the minimum date."""
     date_time = report_date_time()
+    min_date_time = report_date_time("min_report_date")
```

Comment:
What happens when either of these dts is "" instead of an isodt?

Reply:
If one or both are missing you get more measurements. For example, if the min report date is missing there is no lower boundary on the measurements query so basically you get all measurements before the report date. If both are missing you'll get all measurements. But in practice the frontend always passes a min report date.